executor/runtime/docker/docker.go

```diff
@@ -2041,6 +2039,8 @@ func (r *DockerRuntime) k8sContainerToDockerConfigs(c *runtimeTypes.ExtraContain
 		},
 	}
 	if mainContainerRoot != "" {
+		_ = os.MkdirAll(path.Join(mainContainerRoot, "/logs"), 0700)
```

Comment:
I think we should for an error and at least log it. Same with the other `MkdirAll` calls below.

Reply:
Yea I guess. 
These mkdirs are kinda just "best effort", but I suppose it would make for a clearer error code if we couldn't mkdir.
These are temporary. My next PR to make titus-storage handle this will be doing these things (granted it should also handle errors!)